mods/news/init.lua

```diff
@@ -26,13 +26,15 @@ news.chat_reasons.drown = {
 
 minetest.register_privilege("news_report", {give_to_admin = true})
 
-function news.register_deathmsg_tbl(type, name, msgs)
+function news.register_deathmsg_tbl(type, msgs, ...)
     if not (type or name or msgs) then
```

Comment:
This will only return if both `type`, `name` (not defined) and `msgs` are all false (or nil).

Reply:
good catch, forgot to change that line